source/indexes.txt

```diff
@@ -1,8 +1,8 @@
 .. _kotlin-sync-indexes:
 
-=================================
-Optimize Queries by Using Indexes
-=================================
+==============================
+Indexes for Query Optimization
+==============================
```

Comment:
S: suggestion for the Indexes section in general: there isn't much guidance about the Indexes subpages in the spreadsheet, and most drivers just have a single Indexes page without nested pages. I think it's better to avoid L3 nesting in this section, so I moved everything to L2 for PHP. [example](https://deploy-preview-282--docs-php-library.netlify.app/indexes/)